pkg/pillar/evetpm/tpm.go

```diff
@@ -802,6 +853,112 @@ func pcrBankSHA256EnabledHelper() bool {
 	return err == nil
 }
 
+func getTpmEventLogFiles(evtDir string, evtFile string) ([]string, error) {
```

Comment:
Please add comment that this returns all of the sources of event logs from all of the TPM devices.
Also, I wonder if we can use different names for the eventLogFiles (or sourceEventLogFiles) and the savedEventLogFiles - you have constants and variable names which look the same for the origin and the saved.

Reply:
changed the function name, if you still find it unclear I will add a comment.